gsoc2018-bharat/src/CheckPerson.py

```diff
@@ -0,0 +1,48 @@
+import os
+import re
+import sys
+
+def countPronouns(filename, f):
+	"""
+	Takes the input file with the extracted text from the Wikipedia dump
+	and finds whether given article is about a Male or a Female.
+
+	Arguments
+	---------
+	filename : The input file.
+	f : Output file with all the titles and
+	their respective form ( Masculine / Feminine ).
+	"""
+	title = ''
+	wc = 0
+	mascForms = ['he', 'him', 'his']
+	mc = 0
+	femiForms = ['she', 'her', 'hers']
+	fc = 0
+	with open(f, 'w+') as output:
```

Comment:
Tabs are strangely formatted, it's usually 4 spaces (PEP-8 assumes whitespaces instead of tabs for indentation) and many modern IDEs (for example intellij) or text editors (check out the anaconda linter if you use sublime text) can help you with that. variable names should also be snake case `masc_forms`. It also might be useful to isolate the operations you do for a single line here as its own function (taking the line string as input). Your code would be more composable and easier to test.

Reply:
Yes, I will see to it that the PEP-8 standards are met with.